miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/ui/SampleOptionsController.java

```diff
@@ -1,80 +0,0 @@
-package uk.ac.bbsrc.tgac.miso.webapp.controller.rest.ui;
-
-import java.io.IOException;
-
-import javax.servlet.http.HttpServletResponse;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
-import org.springframework.web.util.UriComponentsBuilder;
-
-import uk.ac.bbsrc.tgac.miso.dto.SampleOptionsDto;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.InstituteController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.KitDescriptorRestController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.LabController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.DetailedQcStatusController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.SampleClassController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.SampleGroupController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.SamplePurposeController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.SampleValidRelationshipController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.SubprojectController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.TissueMaterialController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.TissueOriginController;
-import uk.ac.bbsrc.tgac.miso.webapp.controller.rest.TissueTypeController;
-
-@Controller
-@RequestMapping("/rest/ui")
```

Comment:
This is used for the Institute Defaults page.

Reply:
I'm doing this in a separate PR: https://github.com/TGAC/miso-lims/pull/1085